.coafile

```diff
@@ -17,3 +17,7 @@ enabled = False
 files = **/*.rst, README.rst
 ignore = venv/**
 bears = InvalidLinkBear
+
+[rst]
+bears = RSTLintingBear, rstCheckBear
```

Comment:
You have written names of these bears wrong, for correct names please checkout https://github.com/coala/bear-docs#restructuredtext